pkg/apps/controller/deployer/factory.go

```diff
@@ -124,6 +126,15 @@ func (c *DeploymentController) updateReplicationController(old, cur interface{})
 	c.enqueueReplicationController(curRC)
 }
 
+func (c *DeploymentController) addPod(obj interface{}) {
+	pod := obj.(*v1.Pod)
```

Comment:
pod, ok := 
if !ok {}

Reply:
I would call panic in if either way, this calls it as well